help/db_meeting-data.md

```diff
@@ -0,0 +1,39 @@
+### Zawartość tabel
+
+##### Dictionary -- nowe słowniki
+
+| column                   | dictionary_id                        |
+|--------------------------|--------------------------------------|
+| MeetingType.category     | ce12aa23-a5db-49f3-987b-d2db3ab24a3b |
```

Comment:
To są jak rozumiem appendixy do tabeli Column, że deklarujemy nowe kolumny słownikowe, tak? By to mogło być trochę jaśniej napisane.

Reply:
Na pewno słownikiem dla kategorii będzie słownik ce12.., i jest to w tym przypadku bardzo mało customowe powiązanie, ale może rzeczywiście warto skorzystać z tabeli Column, to w sumie prawda, żeby FE miał te wszystkie informacje w jednym miejscu, ok, coś dopiszę